package.json

```diff
@@ -56,5 +56,8 @@
     "prettier": "^1.19.1",
     "start-server-and-test": "^2.0.3",
     "ts-jest": "^28.0.4"
+  },
+  "engines": {
+    "node": ">=18.20.4 <19"
   }
```

Comment:
Tests use node version 16 by default, https://github.com/snapshot-labs/actions/blob/main/.github/workflows/test.yml

if we want to change, we should change test.yml and lint.yml too